app/assets/stylesheets/user_index.scss

```diff
@@ -1,10 +1,29 @@
+@mixin shadow1 {
+  box-shadow: 1px 1px 1px #888888;
+}
+@mixin shadow2 {
+  box-shadow: 2px 2px 2px #888888;
+}
+@mixin shadow3 {
+  box-shadow: 3px 3px 3px #888888;
+}
+@mixin shadow4 {
+  box-shadow: 4px 4px 4px #888888;
```

Comment:
Color literals like `#888888` should only be used in variable declarations; they should be referred to via variable everywhere else.